### PR TITLE
Add nodeName and blockGraffiti to status command

### PR DESCRIPTION
**Pull request by drunkensiberia**
## Summary
FIP: #1858

Adding nodeName and blockGraffiti information to status command via RPC payload.
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by NullSoldier** [comment]
Can you paste the new example output for status so we can see what it looks like in the PR description?

File: ironfish/src/rpc/routes/node/getStatus.ts [hunk 1]
```diff
@@ -211,6 +216,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: 'started',
       miners: node.miningManager.minersConnected,
       blocks: node.miningManager.blocksMined,
+      blockGraffiti: node.config.config.blockGraffiti,
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/node/getStatus.ts:219]: `config.config` is not meant to be used. Use config.ge() instead like this:

```js
config.get('blockGraffiti')
```

File: ironfish/src/rpc/routes/node/getStatus.ts [hunk 2]
```diff
@@ -198,6 +202,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: node.started ? 'started' : 'stopped',
       version: node.pkg.version,
       git: node.pkg.git,
+      nodeName: node.config.config.nodeName,
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/node/getStatus.ts:205]: `config.config` is not meant to be used. Use config.ge() instead like this:

```js
config.get('nodeName')
```

**Comment by drunkensiberia** [comment]
@NullSoldier 

Example with populated nodeName and graffiti:
image


Example with empty nodeName and blockGraffiti:
image

File: ironfish/src/rpc/routes/node/getStatus.ts (hunk 2 above)
Comment (drunkensiberia) [reply ironfish/src/rpc/routes/node/getStatus.ts]: thanks, updated

File: ironfish-cli/src/commands/status.ts [hunk 3]
```diff
@@ -135,6 +141,8 @@ function renderStatus(content: GetStatusResponse): string {
   return `
 Version              ${content.node.version} @ ${content.node.git}
 Node                 ${nodeStatus}
+Node name            ${nodeName}
+Block graffiti       ${blockGraffiti}
```
Comment (drunkensiberia) [inline ironfish-cli/src/commands/status.ts:145]: Maybe we need to change this to:
> Node Name
> Block Graffiti
Does it look better than lowercase n,g?

File: ironfish/src/rpc/routes/node/getStatus.ts [hunk 4]
```diff
@@ -211,6 +216,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: 'started',
       miners: node.miningManager.minersConnected,
       blocks: node.miningManager.blocksMined,
+      blockGraffiti: node.config.get('blockGraffiti').toString(),
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/node/getStatus.ts:219]: Isn't this already a string?

File: ironfish/src/rpc/routes/node/getStatus.ts [hunk 5]
```diff
@@ -198,6 +202,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: node.started ? 'started' : 'stopped',
       version: node.pkg.version,
       git: node.pkg.git,
+      nodeName: node.config.get('nodeName').toString(),
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/node/getStatus.ts:205]: Isn't this already a string?

Comment (drunkensiberia) [reply ironfish/src/rpc/routes/node/getStatus.ts:205]: Yeah they both are strings, that toString() is not necessary here

**Review by NullSoldier — APPROVED** [review]

**Comment by NullSoldier** [comment]
I just noticed that unicode is broken in the blessed box in status, we need to enable it in the screen using this https://github.com/chjj/blessed/issues/377

**Pull request merged**